src/utils/formatCurrencyAmount.ts

```diff
@@ -0,0 +1,21 @@
+import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { V2V3_CURRENCY_ETH, V2V3_CURRENCY_USD } from './v2v3/currency'
+
+/**
+ * Format the input amount with the currency.
+ *
+ * For example, if using `V2V3_CURRENCY_USD` with an amount of 550.1 the format
+ * will be `"$550.1 USD"`.
+ * @returns
+ */
+export const formatCurrencyAmount = ({
+  amount,
+  currency = V2V3_CURRENCY_ETH,
+}: {
+  amount: string | undefined
+  currency: V2V3CurrencyOption | undefined
+}) => {
+  const currencyPrefix = currency === V2V3_CURRENCY_USD ? '$' : ''
```

Comment:
**suggestion** WDYT about using standard currency notation here instead: https://www.investopedia.com/terms/c/currency-symbol.asp#:~:text=If%20the%20currency%20type%20is,such%20as%20the%20Aruban%20Florin.

```suggestion
  const currencyPrefix = currency === V2V3_CURRENCY_USD ? 'US$' : ''
  const currencySuffix = currency === V2V3_CURRENCY_USD ? '' : 'ETH'
```

Or even:

```suggestion
  const currencyPrefix = currency === V2V3_CURRENCY_USD ? 'US$' : 'Ξ'
```

Reply:
I'm happy either way, just unsure if Ξ might trip people up if theyre not familiar with it